optimism/src/lookup.rs

```diff
@@ -114,3 +114,27 @@ pub trait Lookups {
     /// Adds all lookups of Self to the environment
     fn lookups(&mut self);
 }
+
+/// A table of values that can be used for a lookup, along with the ID for the table.
+#[derive(Debug, Clone)]
+pub struct LookupTable<F> {
+    /// The table is a vector of write lookups with the same table ID
+    _table: Vec<Lookup<F>>,
+}
+
+const _TWO_TO_16_UPPERBOUND: u32 = 1 << 16;
```

Comment:
I'd suggest removing this unless it's actually used in several places. Other options:
- Giving it a name that would imply what it's used for. Like `MAX_SRS_SIZE` or `DOMAIN_SIZE_LIMIT`. Not sure why it is a constant actually.
- Moving it into the `LookupTable` implementation.

Reply:
EDIT: I see that in different PRs you actually use this value again. I guess I'd ask to clarify its meaning/rename. Is it the maximum allowed size of the lookup table? Shouldn't there be a constant for that already somewhere on the higher level?